UnitsNet.Tests/CustomCode/AngleTests.cs

```diff
@@ -38,6 +38,8 @@ public class AngleTests : AngleTestsBase
 
         protected override double RevolutionsInOneDegree => 2.777777777777777e-3;
 
+        protected override double TiltInOneDegree => 0.01745240643728351281941897851632;
```

Comment:
Could you point me to a source reference where I can verify this, besides your calculation? I can't seem to find any.

I found these:
https://en.wikipedia.org/wiki/Inclinometer
https://en.wikipedia.org/wiki/Grade_(slope)

Reply:
@bplett-wgtcorp Right, but what I meant is that I can't find any source of reference that says the measurement "tilt" equals Sin(radians) of an angle. A wiki or unit converter or any kind of digital source would be helpful to verify this unit.

Reply:
@bplett-wgtcorp Thanks, no those are plenty!